### DISCUSSION TITLE
[Enhancement] [table] <el-table> 是否可以考虑增加横向拖动

### DISCUSSION THREAD

**Discussion by susu-sama** [element-plus/element-plus#16731] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `table`

## Description

你好，尊敬的element开发团队，el-table功能挺完善的，但为什么不能实现实现横向拖动，有些场景下不想点击滚动条拖动，而是点击<el-table>的任意区域实现电脑端x轴 横向拖动，在移动端中这种是可以的，但是pc端为什么没有这部分， 之前vue2是操作 
this.$refs.myTable.$refs.table.bodyWrapper 去计算  e.clientX的 ，是否可以考虑在el-table增加这个属性或者功能？ 

### Additional comments

_(empty)_

**Comment by tolking** [comment]
shift + 滚轮就可以左右滚动表格。

PC 端的拖动会影响如选中文本等其它操作。

推荐自己实现
